### ISSUE TITLE
Maintenance

### ISSUE THREAD

**Issue by tricoder42** [seikichi/restructured#15]
Hi @seikichi, thank you for creating this library!

I'm just started working on [React writer](https://github.com/rst-js/rst-js) so I can use reStructuredText with GatsbyJS. I see this library isn't complete and several elements aren't implemented. Especially directives would be useful in my case (#11).

If I start sending PRs, do you have time to review them? Or should I rather fork this repository right away?

Thank you again for your time and effort! So far everything works great and I'm looking forward when I switch from Sphinx to GatsbyJS.

**Comment by flying-sheep** [comment]
Seems like this is dead…

**Comment by tricoder42** [comment]
Yep, I've started my parser, but it's abandoned as well https://github.com/rst-js/rst-js

**Comment by cco3** [comment]
:disappointed:

**Comment by Tythos** [comment]
I wouldn't mind submitting a PR or two, as this project is a dependency for one I use quite often (<thoward/rst2html>), and there's a number of parsing features I would love add. But, it does seem like this project is largely inactive now.

**Comment by flying-sheep** [comment]
@seikichi is still active on GitHub. It would be great if they could make contributions possible, maybe by reviewing and merging PRs, maybe by allowing someone to do it.

**Comment by waldyrious** [comment]
FYI, I just released v1.0 of my take on this: https://github.com/waldyrious/rst-playground. Instead of reimplementing a rST parser in JavaScript, I simply run the canonical parser in Python (docutils) directly in the browser via [Pyodide](https://pyodide.org/). Feel free to open issues if you find any shortcomings or have improvement suggestions!

**Comment by flying-sheep** [comment]
ooo that’s a beautiful solution of course!
